src/Arcus.Template.WebApi/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2-stretch-slim AS base
+WORKDIR /app
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2-stretch AS build
+WORKDIR /src
+COPY ["Arcus.Template.WebApi/Arcus.Template.WebApi.csproj", "Arcus.Template.WebApi/"]
+RUN dotnet restore "Arcus.Template.WebApi/Arcus.Template.WebApi.csproj"
+COPY . .
+WORKDIR "/src/Arcus.Template.WebApi"
+RUN dotnet build "Arcus.Template.WebApi.csproj" -c Release
+RUN dotnet new -i "bin/Release/Arcus.Template.WebApi.*.nupkg"
```

Comment:
A NuGet package is created for each template project so we can ship/test independenly.